src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,106 @@
+package homework_2.random_chars_table;
+
+import java.util.Random;
+
+import static homework_2.IOMod.*;
+
+public class RandomCharsTable {
+
+    private static final int RAND_MIN_VALUE = 65; // A
+    private static final int RAND_MAX_VALUE = 90; // Z
+
+    public void run() {
+        inputData();
+    }
+
+    private void inputData() {
+        System.out.print("Input table length, table width, table strategy(even/odd): ");
+        String s = bufferedReaderStringReader();
+
+        try {
+            String[] parameters = s.split(" ");
+            if (parameters.length != 3) {
+                System.out.println(FORMAT_ERROR);
+                return;
+            }
+
+            int tableLength = Integer.parseInt(parameters[0]);
+            int tableWidth = Integer.parseInt(parameters[1]);
+            if (tableLength <= 0 || tableWidth <= 0) {
+                System.out.println(FORMAT_ERROR);
+                return;
+            }
+
+            boolean strategy;
+            if (parameters[2].equalsIgnoreCase("odd")) {
+                strategy = false;
+            } else if (parameters[2].equalsIgnoreCase("even")) {
+                strategy = true;
+            } else {
+                System.out.println(FORMAT_ERROR);
+                return;
+            }
```

Comment:
```suggestion
            boolean strategy = !parameters[2].equalsIgnoreCase("odd");
```
Если будет метод `validInput`, конструкцию можно заменить одной строчкой